imageloading-core/src/main/java/dev/chrisbanes/accompanist/imageloading/MaterialLoadingImage.kt

```diff
@@ -38,13 +38,14 @@ import androidx.compose.ui.geometry.Size
 import androidx.compose.ui.geometry.toRect
 import androidx.compose.ui.graphics.ColorFilter
 import androidx.compose.ui.graphics.ImageAsset
```

Comment:
Remove `ImageAsset` import

Reply:
Weird, ktlint didn't pick this up. Might be something to do with it being a `typealias`

Reply:
Turns out this was because I was using `ImageAsset` is kdoc.